src/msr_data_skx.cpp

```diff
@@ -396,7 +400,8 @@ namespace geopm
                     "units":     "joules",
                     "scalar":    6.103515625e-05,
                     "writeable": false,
-                    "aggregation": "sum"
+                    "aggregation": "sum",
+                    "description": "Actual energy used."
```

Comment:
increasing meter of energy used over time.  It will reset periodically due to roll-over.

Reply:
...used of the PKG over time. ...